src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE = 40;
+  const MIDDLE_TERM_DISCOUNT = 20;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3
+  const LONG_TERM = 7
+
+  if (days < SHORT_TERM) {
+    return days * PRICE;
```

Comment:
You can create variable for base price and use it everywhere 
```suggestion
  const LONG_TERM = 7;
  const BASE_PRICE = days * PRICE;

  if (days < SHORT_TERM) {
    return BASE_PRICE;
```